test/noAutoReload.js

```diff
@@ -45,9 +71,9 @@ describe('Apostrophe-i18n-static', function() {
 
       // even after visiting a template, JSON is not generated due to "autoReload: false" option in apos
       // see configuration in appWithoutAutoReload.js
-      await rp('http://localhost:3000');
+      await rp('http://localhost:9999');
       const file = JSON.parse(await asyncReadFile('./test/locales/en-US.json', { encoding: 'utf8' }));
-      expect(file).to.have.property('test2', 'test');
+      expect(file).to.not.have.property('test2', 'test');
```

Comment:
This test should be "expect(file).to.have.property('test2', 'test');" because we test the ability to keep the same translation when not reloading the JSON file (even if the translation has changed in the DB).

Reply:
This was caused by my unnecessary hack that @daklik pointed out. I have backed that change out and restored the test your way. It passes now.